x-pack/legacy/plugins/ml/public/application/management/index.ts

```diff
@@ -22,20 +22,21 @@ if (
   xpackInfo.get('features.ml.showLinks', false) === true &&
   xpackInfo.get('features.ml.licenseType') === LICENSE_TYPE.FULL
 ) {
-  management.register('ml', {
-    display: i18n.translate('xpack.ml.management.mlTitle', {
-      defaultMessage: 'Machine Learning',
-    }),
-    order: 100,
-    icon: 'machineLearningApp',
-  });
-
-  management.getSection('ml').register('jobsList', {
-    name: 'jobsListLink',
-    order: 10,
-    display: i18n.translate('xpack.ml.management.jobsListTitle', {
-      defaultMessage: 'Jobs list',
-    }),
-    url: `#${JOBS_LIST_PATH}`,
-  });
+  // @ts-ignore
+  npStart.plugins.management.legacy
```

Comment:
Is this `ts-ignore` because `legacy` has been given `unknown` type in `ManagementStart`?

Reply:
will take a look

Reply:
They were, thanks for the nudge